src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -1,15 +1,91 @@
 """
-Download genome files from NCBI FTP server
-"""
+PROTOTYPE - Download genome files from NCBI FTP server.
+
+usage: ncbi_downloader.py [-h] --download_file_ext
+                          DOWNLOAD_FILE_EXT
+                          [DOWNLOAD_FILE_EXT ...]
+                          --release_ver
+                          {202,207,80,83,86,89,95}
```

Comment:
This is going to change over time, should probably be removed unless I'm missing something

Reply:
👍

Reply:
:+1: